docs/lambda-k-pi/index.md

```diff
@@ -0,0 +1,14 @@
+# pγ → ΛK⁺π⁰
+
+We are doing amplitude analysis and Partial Wave Analysis (PWA) with [CAS-assisted model building](https://compwa.github.io/symbolics), which is a techniqued employed in the ComPWA project.
+
+- The first demonstration example is:<br>
+  $p \gamma \to$ Resonances (e.g. N\*) $\to \eta \pi^0 p$
+- The main focus of this research would be:<br>
+  $p \gamma \to$ Resonances (e.g. N\*, K\*) $\to K^+ \pi^0 \Lambda$
```

Comment:
Why is this repeating what is already on the main page?

Reply:
ok